src/info.h

```diff
@@ -6,6 +6,9 @@
 
 #include "font.h"
 
+// Config key for info block timeout
+#define VIEWER_CFG_INFO_TIMEOUT "timeout"
```

Comment:
It is info's object, so `INFO_CFG_TIMEOUT` and it would be better to move it to `info.c`.